compose-seed.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+cd $(dirname $0)
+set -e
+
+wait_secret() {
+    token_count="0"
+    for i in $(seq 1 120); do
+        token_count=$(docker exec hf-k3d kubectl get secret -o name | grep "$1" | wc -l)
+        if [ "$token_count" != "0" ]; then
+            break
+        fi
+        echo "waiting for $1 to be ready: ${i}s"
+        sleep 1
+    done
+    if [ "$token_count" = "0" ]; then
+        echo "$1 not ready after 120 seconds" >&2
+        exit 1
+    fi
+}
+
+# wait for default namespace to be ready
+wait_secret "default-token"
+
+# apply CRDs
+docker exec hf-k3d \
+    kubectl apply -f /app/charts/hobbyfarm/crds/
+
+# apply local seed data
+docker exec hf-k3d \
+    kubectl apply -f /app/cicd/seed-local/
```

Comment:
Can we combine the 2 "seed data" locations? I don't care what the content is but no point in having this _and_ `charts/hobbyfarm/templates/seed`. We should probably keep it in the chart though so it can still be deployed with dummy data.

Reply:
not easily, the chart has helm templating in it's resources.  kubetl apply does not understand.

Reply:
Good point -- we'd have to run `helm template` on that directory.